src/main/java/Main.kt

```diff
@@ -14,41 +12,32 @@ import tree.ImpTargetTreeNode
 import tree.TargetTreeNode
 
 object Main {
+
+    private const val jFrameTitle = "Antlr AST"
+
     @Throws(Exception::class)
     @JvmStatic
     fun main(args: Array<String>) {
-        println(CharStreams.fromFileName("test.tp"))
-        val lexer = TreepatLexer(CharStreams.fromFileName("test.tp"))
+        val lexer = TreepatLexer(CharStreams.fromFileName(args.first()))
```

Comment:
Nice use of arguments, it's cleaner and should make easier the development of new functionalities.